doc/memos/rdm-draft-alamos-ieee802154-radio-hal.md

```diff
@@ -322,121 +327,57 @@ function to enable the radio device, if the initialization procedure succeeded.
+### 5.2.1 State specification
+A specification for each state is described in the following items.
 
-### 5.2.1 Off
+- OFF: If radio initialization succeeds, the Abstract State Machine begins in
+  `OFF` state. During this state the device consumes the least power and all
+  hardware components (transceiver, crypto acceleration) are disabled.
 
-If radio initialization succeeds, the Abstract State Machine begins in `Off`
-state. During this state the device consumes the least power and all hardware
-components (transceiver, crypto acceleration) are disabled.
+- TRX OFF: During this state the device is on but the transceiver is off (PLL
+  not locked). The power consumption is higher than the `OFF` state but the
+  device is able to operate the transceiver and other hardware components (e.g
+  crypto accelerator).
 
-Calling the `on` function request a state change to `TRX_OFF` state. The
-transition finishes when the `on` function returns 0 or the Event Notification
-reports `RF_AWAKE_END`.
+- IDLE: This state is device specific and represents a state where the device
+  is ready to transmit, fetch a received frame, change the Physical Information
+  Base or perform Stand-Alone CCA.
 
-### 5.2.2 TRX OFF
+- RX ON: During RX ON state the radio is able to detect Start Frame Delimiter
+  (SFD) and thus, receive frames.
 
-During this state the device is on but the transceiver is off (PLL not locked).
-The power consumption is higher than the `Off` state but the device is able
-to operate the transceiver and other hardware components (e.g crypto
-accelerator).
-
-Setting the transceiver state to IDLE (`set_trx_state`) leaves the `TRX_OFF`
-state. The transition finishes when the Event Notification reports
-`RF_PLL_LOCK`.
-
-### 5.2.3 Idle
-
-This state is device specific and represents a state where the device is ready
-to transmit, fetch a received frame or perform Stand-Alone CCA.
-
-Assuming a frame is already loaded in the framebuffer, calling `transmit()` will
-set the state to `TX_BUSY` and begin the transmission of a frame. The Abstract
-State Machine will go back to Idle state on either `RF_TX_DONE` event or
-`RF_CCA_FAIL`, if using Auto-CCA and the channel was busy.
-
-A Stand-Alone CCA can be requested with `cca()`. The Abstract State Machine
-will change state to `CCA_BUSY` and go back to Idle state on `RF_CCA_IDLE` or
-`RF_CCA_FAIL` events.
-
-Finally, setting the transceiver state to `TRX_OFF` or `RX_ON` will leave the
-Idle state (this will generate an `PLL_UNLOCK` event in the former).
-
-### 5.2.4 RX ON
-
-During RX ON state the radio is able to detect Start Frame Delimiter (SFD) and
-thus, receive frames.
-
-The state is left when an SFD is detected or the transceiver state is set to
-Idle. The former will set the Abstract State Machine state to the transient
-state RX BUSY and then to Idle on `RF_RX_DONE` event. The latter will change
-the state to Idle only if the radio is currently not receiving a frame.
+TODO: Calls
```

Comment:
I guess this sneaked in by accident? Having a TODO in the final document does not make it very final.

Reply:
oh, good catch. I forgot that

Reply:
done!